src/nextjs/nextjs-wizard.ts

```diff
@@ -1071,6 +1117,40 @@ async function askShouldSetTunnelRoute() {
   });
 }
 
+/**
+ * Ask users if they want to create a .sentryrules file with AI rule examples for Sentry.
+ * This is useful for giving the LLM context on common actions in Sentry like custom spans,
+ * logging, and error / exception handling.
+ */
+
+async function askShouldCreateAiRulesFile(): Promise<boolean> {
+  return await traceStep('ask-create-ai-rules-file', async (span) => {
+    const shouldCreateAiRulesFile = await abortIfCancelled(
+      clack.select({
+        message:
+          'Do you want to create a ./rules/sentryrules.md file with AI rule examples for Sentry?',
```

Comment:
here we should add `chalk.cyan` around the filename

```suggestion
          'Do you want to create a ./rules/sentryrules.md file with AI rule examples for Sentry?',
```